README.md

```diff
@@ -95,16 +97,16 @@ head.link
 head.script
 ```
 
-`head` contains five possible properties, `title`, `base`, `meta`, `link`, `script`:
+`head` contains five possible properties, `htmlAttributes`, `title`, `base`, `meta`, `link`, `script`:
 
-- Each property contains `toComponent()` and `toString()` methods. Use whichever is appropriate for your environment. E.g:
+- Each property contains `toComponent()` and `toString()` methods. Use whichever is appropriate for your environment. For htmlAttributes, use the JSX spread operator on the object returned by `toComponent()`. E.g:
```

Comment:
It's actually a JSX thing: https://facebook.github.io/react/docs/jsx-spread.html#spread-attributes
They copied ES6 syntax to keep it simple but it serves a slightly different purpose